File: Object_Orianted_Programming_For_YTM/Java/AbstractClassExample/src/Daire.java
public class Daire extends Sekil{
    double yariCap;
    
    public Daire(double yariCap) {
        this.yariCap = yariCap;
    }
    
    double alanHesapla() {
        return Math.PI * yariCap * yariCap;
    }
}

File: Object_Orianted_Programming_For_YTM/Python/AbstractClassExample/src/Daire.py
from Sekil import Sekil
class Daire(Sekil):
    def __init__(self, yari_cap):
        self.yari_cap = yari_cap
    
    def alan_hesapla(self):
        return 3.14 * self.yari_cap * self.yari_cap